Automattic/wp-calypso
# Refactor licensing auto-activation UI

#### Changes proposed in this Pull Request

By applying the `<LicensingActivation />` component that was introduced in #57649, we can clean up the JSX and SCSS styles quite a bit.

**Note:** This is just cleaning things up in licensing checkout thank-you component and styles. No changes to Checkout logic or post-checkout redirects were made in this PR. 👍

Related to 1201096622142517-as-1201324181136912

#### Testing instructions

- Download this PR.
- Start Calypso with `yarn start`.
- Visit http://calypso.localhost:3000/checkout/jetpack/thank-you/licensing-auto-activate/jetpack_backup_daily.
- Verify it looks like the figma mockup: ceKxiVna5wfpmwFBsojrAx-fi-2219%3A25549 
- Verify the code changes and SCSS changes look reasonable.

Screenshot:

![Screenshot on 2021-11-05 at 10-12-26](https://user-images.githubusercontent.com/11078128/140524200-81cdda41-9f03-4d43-a8cd-833ed0e8c4a8.png)
